api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+            items:
+              $ref: '#/definitions/Partition'
+  '/topics/{topic}/partitions/{partition}':
+    get:
+      tags:
+        - low-level-api
+      summary: Returns the given `Partition` of this topic
+      parameters:
+        - name: topic
+          in: path
+          description: Topic name
+          type: string
+          required: true
+        - name: partition
+          in: path
+          description: Partition id.
+          type: string
+          required: true
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Partition'
+  '/topics/{topic}/partitions/{partition}/events':
+    post:
+      tags:
+        - low-level-api
+      summary: |
+        Posts a batch of `Event`s to the specified partition of this topic.
+        Note that this overwrites the evaluation of the partitioning_key.
+      parameters:
+        - name: topic
+          in: path
+          description: Topic where to send events to.
+          type: string
+          required: true
+        - name: partition
+          in: path
+          description: Partition where which the events are stored.
+          type: string
+          required: true
+        - name: Content-Encoding
+          in: header
+          required: false
+          description: '"Allow the publisher to gzip compress the payload by setting this header to "gzip". For Example "Content-Encoding: gzip"'
+          type: string
+        - name: event
+          in: body
+          description: Array of `Event`s being sent.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Event'
+      responses:
+        '201':
+          description: Event submitted.
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Not allowed
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+    get:
+      tags:
+        - low-level-api
+      summary: |
+        Starts a stream delivery for the specified partition of the given topic. The events are send in batches.
+        The tracking of the current position in the partition and of which
+        partition is being read is in the responsibility of the client.
+        No commits are needed.
+      parameters:
+        - name: topic
+          in: path
+          description: Topic to consume events from
+          type: string
+          required: true
+        - name: partition
+          in: path
+          description: Id of the partition where to get events from
+          type: string
+          required: true
+        - name: start_from
```

Comment:
editorial: "start_offset" is a better name -- API handles offsets of partitions.